src/libs/Visibility/index.js

```diff
@@ -7,12 +8,18 @@ import ELECTRON_EVENTS from '../../../desktop/ELECTRON_EVENTS';
  * so we ask the main process synchronously whether the
  * BrowserWindow.isFocused()
  *
+ * However, when the AppState is active, we want to use document.visibilityState.
+ * For example, when expanding the window after minimizing, BrowserWindow.isFocused()
+ * is false even if it is fully visible in the foreground.
```

Comment:
Can you include more details on why we use document.visibilityState here?

> BrowserWindow.isFocused() is false even if it is fully visible in the foreground

Details if this is a bug / or why we are no longer using this

Reply:
Will do. This got me thinking about the reasoning a bit more, and I'm not entirely sure this makes sense. I might need to make a slight adjustment to the logic, but I'll get back to you.

Reply:
**random commenter:** 

I think we are also mixing the web and desktop implementations a bit here and should probably split these into a:

`index.website.js`
`index.desktop.js`

files instead of using `window.electron` (which I assume is always present in the electron app).

Would be great if we can manage that cleanup here and I think will make the code a bit more readable.